Source/MLXNN/Module.swift

```diff
@@ -392,6 +392,7 @@ open class Module {
     ///
     /// This passes `verify: .none`.  Note that there may still be `fatalErrors()` if
     /// for example an `MLXArray` is set on a `Module`.
+    @discardableResult
     public func update(parameters: ModuleParameters) -> Self {
```

Comment:
Without this annotation a caller that ignores the result will get a warning.  These are really meant for chaining calls and are completely optional.